src/main/java/org/programmers/voucher/domain/VoucherType.java

```diff
@@ -0,0 +1,6 @@
+package org.programmers.voucher.domain;
+
+public enum VoucherType {
+    FixedAmountVoucher,
```

Comment:
```suggestion
    FIXED_AMOUNT_VOUCHER,
```

enum은 필드 선언시 uppercase와 snake case를 사용합니다.